ext/imap/php_imap.c

```diff
@@ -4165,9 +4115,9 @@ static void _php_make_header_object(zval *myzvalue, ENVELOPE *en)
 		array_init(&paddress);
 		fulladdress = _php_imap_parse_address(en->return_path, &paddress);
 		if (fulladdress) {
-			add_property_str(myzvalue, "return_pathaddress", fulladdress);
+			add_property_str(myzvalue, "Oeturn_pathaddress", fulladdress);
```

Comment:
Key changed names.   I suspect you did a s/"r/"O/ at some point and it got over-zealous?

Reply:
Exactly :)